src/FixMixedTabs/Impl/MarginFactory.cs

```diff
@@ -18,16 +18,16 @@ namespace FixMixedTabs
     internal sealed class MarginFactory : IWpfTextViewMarginProvider
     {
         [Import]
-        private ITextDocumentFactoryService _TextDocumentFactoryService = null;
-        [Import]
+        private ITextDocumentFactoryService _TextDocumentFactoryService;
```

Comment:
I'm getting error CS0649: Field 'MarginFactory._TextDocumentFactoryService' is never assigned to, and will always have its default value null.
That's a warning, but this project is setup to treat warnings as errors.

Is it building OK for you? Unfortunately appveyor is still not functioning.